pkg/handler/repository_parameters_test.go

```diff
@@ -10,15 +10,17 @@ import (
 	"testing"
 
 	"github.com/content-services/content-sources-backend/pkg/api"
+	"github.com/content-services/content-sources-backend/pkg/config"
 	"github.com/content-services/content-sources-backend/pkg/dao"
-	"github.com/labstack/echo/v4"
 	"github.com/openlyinc/pointy"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 )
 
 func serveRepositoryParametersRouter(req *http.Request, mockDao *MockRepositoryDao) (int, []byte, error) {
-	router := echo.New()
+	// TODO use echo.New() and add to the context the key where
+	//      the middleware store the x-rh-identity structure
```

Comment:
are you planning on addressing these TODOs as part of this?  If not, lets move it to an open task in jira

Reply:
I was at the beginning; but looking at again, I have updated the code to add only what it requires for the test.